src/components/LiveFeedsSlide.jsx

```diff
@@ -45,49 +74,309 @@ const CloseButton = styled(Button)(({ theme }) => ({
+          minPrice: 23.0,
+          bidAmount: 12032.25,
+          seller: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+          bidder: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+          currency: 'ZOOM',
+          winner: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+        }
+      })
+    )
+
+    dispatch(
+      Actions.myNewBidEventTriggered({
+        type: 'settlemybid',
+        timestamp: Date.now() / 1000,
+        content: {
+          blockNumber: Date.now(),
+          // itemNumber: itemNumber.toNumber(),
+          itemNumber: 10,
+          minPrice: 23.0,
+          bidAmount: 12032.25,
+          seller: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+          bidder: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+          currency: 'ZOOM',
+          winner: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+        }
+      })
+    )
+
+    dispatch(
+      Actions.newBidEventTriggered({
+        type: 'settled',
+        timestamp: Date.now() / 1000,
+        content: {
+          blockNumber: Date.now(),
+          // itemNumber: itemNumber.toNumber(),
+          itemNumber: 10,
+          minPrice: 23.0,
+          bidAmount: 12032.25,
+          seller: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+          bidder: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+          currency: 'ZOOM',
+          winner: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+        }
+      })
+    )
+
+    dispatch(
+      Actions.myNewBidEventTriggered({
+        type: 'win',
+        timestamp: Date.now() / 1000,
+        content: {
+          blockNumber: Date.now(),
+          // itemNumber: itemNumber.toNumber(),
+          itemNumber: 10,
+          minPrice: 23.0,
+          bidAmount: 12032.25,
+          seller: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+          bidder: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+          currency: 'ZOOM',
+          winner: '0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232',
+        }
+      })
+    )
+  }
+
+
+  const _remoteThisFUNCTION = () => {
+    const itemListedEvent = {
+      itemNumber: 42,
+      auctionEnd: Date.now() / 1000,
+      tokenIds: [1,2,3],
+      minPrice: 10.5,
+      highestBid: 0,
+      bidder: "0x24213bd4cEc78A8843B50b9503c1d56eEA4d0232",
+      saleToken: '',
+      currency: 'ZOOM',
+      nftToken: "nfttoken",
+      auctionStart: moment().unix(),
+      highestBidder: null,
+    }
+
+    const itemListedEvent1 = {
+      itemNumber: 42,
+      auctionEnd: Date.now() / 1000,
+      tokenIds: [1,2,3],
+      minPrice: 10.5,
+      highestBid: 0,
+      bidder: "0x24213bd4cEc78A8843B50b9503c1d56eEA4d0231",
+      saleToken: '',
+      currency: 'ZOOM',
+      nftToken: "nfttoken",
+      auctionStart: moment().unix(),
+      highestBidder: null,
+    }
+
+    PubSub.publish(EVENT_TYPES.Bid, itemListedEvent);
+    PubSub.publish(EVENT_TYPES.Bid, itemListedEvent1);
+  }
+
+  const { data: generalAlerts } = useFetchLiveFeeds("General")
```

Comment:
nit, you don't actually need to set up a hook for getting the live feeds. You can replace this with `queryClient.getQueryData(queryKey)` for each type of filter